app/proguard-project.txt

```diff
@@ -27,4 +27,7 @@
 # in the projects, so the classes aren't present.  These warnings are safe to suppress.
 -dontwarn kotlinx.serialization.KSerializer
 -dontwarn kotlinx.serialization.Serializable
--dontwarn kotlinx.serialization.internal.AbstractPolymorphicSerializer
\ No newline at end of file
+-dontwarn kotlinx.serialization.internal.AbstractPolymorphicSerializer
+
+-keep class java.util.concurrent.*
```

Comment:
Could we be more specific here to avoid keeping all the underlying classes? Or could we get rid of the warnings by `dontwarn`? Or is this the only working way? Thanks for the explanation.